raiden/network/pathfinding.py

```diff
@@ -287,7 +287,8 @@ def configure_pfs_or_exit(
         sys.exit(1)
 
     if not is_same_address(
-        pathfinding_service_info.token_network_registry_address, token_network_registry_address
+        Address(pathfinding_service_info.token_network_registry_address),
```

Comment:
Instead of downcasting `pathfinding_service_info.token_network_registry_address` to `Address` maybe change the function argument from `token_network_registry_address: Address,` to `token_network_registry_address: PaymentNetworkAddress,`?